src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -28,6 +28,7 @@ class DynamoCoreSetup
         private readonly HostAnalyticsInfo analyticsInfo;
         private const string sandboxWikiPage = @"https://github.com/DynamoDS/Dynamo/wiki/How-to-Utilize-Dynamo-Builds";
         private DynamoView dynamoView;
+        private IDSDKManager idsdkManager;
```

Comment:
I'm confused by this, don't you create *another* instance of this object in the makeModel function?

Reply:
MakeModel do not get triggered until we launch dynamo from the splash screen, this instance of IDSDKManager, is used to login before loading Dynamo from splash screen.

Reply:
I think I can make this work with the same instance, updating soon

Reply:
Notice I am also going to move Splash Screen into Dynamo View instead of Sandbox, so your PR may also change?